models/user.go

```diff
@@ -236,6 +236,12 @@ func (u *User) GetEmail() string {
 	return u.Email
 }
 
+// GetAllUsers returns a slice of all users found in DB.
+func GetAllUsers() ([]*User, error) {
+	users := make([]*User, 0)
+	return users, x.OrderBy("id").Find(&users)
+}
```

Comment:
```suggestion

type AllUsersOpts struct {
	AdminsOnly bool
}

// GetAllUsers returns a slice of all users found in DB.
func GetAllUsers(opts AllUsersOpts) ([]*User, error) {
	users := make([]*User, 0)
	return users, x.OrderBy("id").Where("is_admin = ?", opts.AdminsOnly).Find(&users)
}

```

Reply:
Hi, it seems this change breaks the build (I don't know how didn't see this before :/)
```# code.gitea.io/gitea/cmd
cmd/admin.go:406:34: not enough arguments in call to models.GetAllUsers
	have ()
	want (models.AllUsersOpts)
```
Could you help please ?